### PR TITLE
Adding Chapter Placeholders

### PR DESCRIPTION
**Pull request by johko**
Placeholders for the chapters, according to the structure from the spreadsheet (https://docs.google.com/spreadsheets/d/1fjmbsdGwe7IUMBv74LDC7IpoJy8ijiFdzGdnDlBv6eA/edit#gid=0)

I also added a README.md for each section, because I think it is nice to have little written overview and not just the notebooks in the folders.

### CODE REVIEW

**Comment by notvicent3** [comment]
> Placeholders for the chapters, according to the structure from the spreadsheet (https://docs.google.com/spreadsheets/d/1fjmbsdGwe7IUMBv74LDC7IpoJy8ijiFdzGdnDlBv6eA/edit#gid=0)
> 
> I also added a README.md for each section, because I think it is nice to have little written overview and not just the notebooks in the folders.

This is amazing, GJ @johko  <3

File: course/4 Computer Vision in the Wild/4.3 Ethics and Biases/README.md
Comment (lunarflu) [inline course/4 Computer Vision in the Wild/4.3 Ethics and Biases/README.md:1]: 💖Great to have a section like this!

**Review by lunarflu — APPROVED** [review]
Looks great to me! A lot more detailed. cc @merveenoyan in case she has any extra ideas for stuff to cover 🤗

File: course/4 Computer Vision in the Wild/4.3 Ethics and Biases/README.md
Comment (notvicent3) [reply course/4 Computer Vision in the Wild/4.3 Ethics and Biases/README.md:1]: > 💖Great to have a section like this!

A must nowadays looking how it's all evolving =)

**Pull request merged**